### ISSUE TITLE
which dataset did you use?

### ISSUE THREAD

**Issue by dypromise** [taotaonice/FaceShifter#21]
Hi, taotonice
Nice work! paper author said he use CelebHQ\FFHQ\VGGface dataset, and  did not tell about some details about dataset. what dataset did you use? and since you have did some experiments,  could you give some suggestions or experience about trainning? thank you very much

**Comment by taotaonice** [comment]
About dataset, just as the paper said is OK. At the beginning of training, it is suggested to remove the GAN loss.

**Comment by ocastan** [comment]
I've replaced visdom with tensorboard to monitor the various loss and the discriminator scores. I noticed once the discriminator starts discriminating real from fake the generator can't compensate and does not learn anymore. To prevent this I update the discriminator every 10 iterations with good results so far.

**Comment by taotaonice** [comment]
thank you for your discovery

**Comment by ocastan** [comment]
And I've added D.requires_grad_(False) before G gradient calculation and D.requires_grad_(True) before D gradient calculation.

**Comment by dypromise** [comment]
> ate the discriminator every 10 iterations

could you show some results? I use FFHQ only, resized to 256x256, then feed the attrNet whole image, feed the faceIDNet the croped image based on MTCNN crop rule, but I found that after 30epoch the result preserving ‘bang' in src image, in other words, I can't produce clear result like author or this repo. About face crop, I try many options, when crop region is so small, faceID net can not work well.

**Comment by ocastan** [comment]
I use VGGface and celebaHQ with a min face size of 200 and pictures with size less than 8k removed. With discriminator update 10 times slower, the discriminator overcome the generator after 30k iterations instead of 10-12k. I'm looking for the best way to calculate L_attr: with Y gradient only, with Xt gradient only and both. No perfect results so far...

**Comment by PMA65** [comment]
@ocastan could you please share your email

**Comment by ocastan** [comment]
@PMA65 why not, can you tell me more?

**Comment by PMA65** [comment]
@ocastan Thanks for responding, I'll tell you all details at soulak.9384@gmail.com
please contact as soon as you see this, thank you

**Comment by ocastan** [comment]
@PMA65 I can't understand why we should leave this public forum and use a private channel to exchange
